src/components/ListView/index.jsx

```diff
@@ -1,9 +1,32 @@
 import React from 'react';
+import { useCollection } from 'react-firebase-hooks/firestore';
+import { getProducts } from 'components/Utils/firestore';
 
 function ListView() {
+  const query = getProducts();
+  const [values, loading, error] = useCollection(query);
+
   return (
     <div>
       <h2>List</h2>
+      <div>
+        {error && <strong>Error: {JSON.stringify(error)}</strong>}
+        {loading && <span>List: Loading...</span>}
+        {values && (
+          <span>
+            <div>
+              {values.docs.map((doc) => (
+                <div key={doc.id}>
+                  <p>
+                    {doc.data().item.charAt(0).toUpperCase() +
+                      doc.data().item.slice(1)}
```

Comment:
This is a bit strange to read, moving forward I suggest you to abstract that functionality into a constant/function with a meaningful name.

For instance, 

```js
// It doesn't have to be `itemName`, but I hope you get the idea 😅
const itemName = doc.data().item.charAt(0).toUpperCase() + doc.data().item.slice(1)

// later in the render function
<p>{itemName}</p>
```